File: filedata/entity/java/vipRecord.java

package entity;
// 
public class Viprecord  extends BaseDomain{
    public Viprecord() {

    }
    
    //
    private String Id;
        
    //
    private String CompanyStyle;
        
    //
    private String UserName;
        
    //
    private bit(10) Status;
        
    //
    private Date CreatTime;
        
    //
    private Date StartTime;
        
    //
    private Date EndTime;
        
    //
    private int(10) OpenType;
        
    //
    private String Remark;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public String getUsername() {
        return Username;
    }
    public void setUsername(String Username) {
        this.Username = Username;
    }
            
    public bit(10) getStatus() {
        return Status;
    }
    public void setStatus(bit(10) Status) {
        this.Status = Status;
    }
            
    public Date getCreattime() {
        return Creattime;
    }
    public void setCreattime(Date Creattime) {
        this.Creattime = Creattime;
    }
            
    public Date getStarttime() {
        return Starttime;
    }
    public void setStarttime(Date Starttime) {
        this.Starttime = Starttime;
    }
            
    public Date getEndtime() {
        return Endtime;
    }
    public void setEndtime(Date Endtime) {
        this.Endtime = Endtime;
    }
            
    public int(10) getOpentype() {
        return Opentype;
    }
    public void setOpentype(int(10) Opentype) {
        this.Opentype = Opentype;
    }
            
    public String getRemark() {
        return Remark;
    }
    public void setRemark(String Remark) {
        this.Remark = Remark;
    }
             
    @Override
    public String toString() {
        return "vip_record[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",UserName=" + UserName + ",Status=" + Status + ",CreatTime=" + CreatTime + ",StartTime=" + StartTime + ",EndTime=" + EndTime + ",OpenType=" + OpenType + ",Remark=" + Remark + "]";
    }
        
    
}
        

File: filedata/entity/c#/Viprecord.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("vip_record")]
    public class Viprecord
    {
        
        /// <summary>
        /// ID主键
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 会员站点
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 会员账号
        /// </summary>
        [Column("UserName")]
        public string UserName { get; set; }
    
    
        /// <summary>
        /// 状态 0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public bool Status { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreatTime")]
        public DateTime CreatTime { get; set; }
    
    
        /// <summary>
        /// 开始时间
        /// </summary>
        [Column("StartTime")]
        public DateTime StartTime { get; set; }
    
    
        /// <summary>
        /// 结束时间
        /// </summary>
        [Column("EndTime")]
        public DateTime EndTime { get; set; }
    
    
        /// <summary>
        /// 开通类型 0=未知 1=购买VIP开通
        /// </summary>
        [Column("OpenType")]
        public int OpenType { get; set; }
    
    
        /// <summary>
        /// 备注
        /// </summary>
        [Column("Remark")]
        public string Remark { get; set; }
    
    
    }
}
